src/components/HTML/HTML.tsx

```diff
@@ -1,35 +1,61 @@
-import React from 'react';
+import React, {PropsWithChildren, useMemo} from 'react';
 
-import {QAProps} from '../../models/common';
-import {hasBlockTag} from '../../utils';
+import {ClassNameProps, QAProps} from '../../models/common';
+import {selectVariant} from '../../utils';
 
-export interface HTMLProps {
-    children?: string;
+export interface HTMLExtraProps {
+    variant?: 'span' | 'div' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'section' | 'p';
```

Comment:
would it be easier to make an enum for variand use it in other places like this https://github.com/gravity-ui/page-constructor/pull/993/files#diff-8a573455e8636f1d6dd641125e9b47de96bbf75f3fdd9bff39a2233e7515144bR27 ?

Reply:
enum is good if value is unknown, or difficult to write. When value is simple and well-known it is better to use simple type as I do.